person/tests.py

```diff
@@ -0,0 +1,876 @@
+            name="Tomas2",
+            surname='Camero',
+            birth_date=datetime.datetime(2000, 1, 24, tzinfo=datetime.timezone.utc),
+            sex="M",
+            city="Sevilla",
+            address="Logroño 19",
+            telephone=691644398,
+            postal_code=41730,
+            condition="EM",
+            member="UNA",
+            user_type="OTROS",
+            correspondence="CC",
+            status="F",
+            family_unit_size=1,
+            own_home=None,
+            own_vehicle=False,
+            bank_account_number="ES6700832134418939683449")
+                
+    def test_asem_user_create_own_vehicle_incorrect(self):
+        with self.assertRaises(Exception):
+            ASEMUser.objects.create(email="tcamerob2@gmail.com",
+            name="Tomas2",
+            surname='Camero',
+            birth_date=datetime.datetime(2000, 1, 24, tzinfo=datetime.timezone.utc),
+            sex="M",
+            city="Sevilla",
+            address="Logroño 19",
+            telephone=691644398,
+            postal_code=41730,
+            condition="EM",
+            member="UNA",
+            user_type="OTROS",
+            correspondence="CC",
+            status="F",
+            family_unit_size=1,
+            own_home="VP",
+            own_vehicle="No es un booleano",
+            bank_account_number="ES6700832134418939683449")
+                
+    def test_asem_user_create_own_vehicle_incorrect_null(self):
+        with self.assertRaises(Exception):
+            ASEMUser.objects.create(email="tcamerob2@gmail.com",
+            name="Tomas2",
+            surname='Camero',
+            birth_date=datetime.datetime(2000, 1, 24, tzinfo=datetime.timezone.utc),
+            sex="M",
+            city="Sevilla",
+            address="Logroño 19",
+            telephone=691644398,
+            postal_code=41730,
+            condition="EM",
+            member="UNA",
+            user_type="OTROS",
+            correspondence="CC",
+            status="F",
+            family_unit_size=1,
+            own_home="VP",
+            own_vehicle=None,
+            bank_account_number="ES6700832134418939683449")
+                
+    def test_asem_user_create_bank_account_number_incorrect_max(self):
+        with self.assertRaises(Exception):
+            ASEMUser.objects.create(email="tcamerob2@gmail.com",
+            name="Tomas2",
+            surname='Camero',
+            birth_date=datetime.datetime(2000, 1, 24, tzinfo=datetime.timezone.utc),
+            sex="M",
+            city="Sevilla",
+            address="Logroño 19",
+            telephone=691644398,
+            postal_code=41730,
+            condition="EM",
+            member="UNA",
+            user_type="OTROS",
+            correspondence="CC",
+            status="F",
+            family_unit_size=1,
+            own_home="VP",
+            own_vehicle=True,
+            bank_account_number="ES11111111111111111111111")
+    
```

Comment:
You could do a test that does not meet the pattern established for the bank account.

Reply:
okey